lib/store.ts

```diff
@@ -28,15 +28,37 @@ export type AppStore = ReturnType<typeof makeStore>
 export type RootState = ReturnType<AppStore['getState']>
 export type AppDispatch = AppStore['dispatch']
 
-export const selectRecommendStandardDeduction = (state: RootState) => {
+/**
+ * Post-expenses, evaluate if standard deduction is more avantageous. 
+ * If total income * std ded > expenses, then should return true
+ * 
+ * TODO: Needs tests
+ * TODO: why are the std ded methods here and not in benefits slice?
+ * 
+ * @param state 
+ * @returns 
+ */
+export const isStandardDeductionBetter = (state: RootState) => {
     const benefits = selectBenefits(state)
     const incomeTotal = selectIncomeTotal(state)
     const expenseTotal = selectExpenseTotal(state)
 
-    if (!benefits.snap || benefits.deductionAmount === undefined) {
-        return false
-    }
-
     const percent = benefits.deductionAmount / 100
     return expenseTotal < incomeTotal * percent
+}
+
+/**
+ * Do we recommend they take a standard deduction based on SNAP?
+ * This is to evaluate before they add expenses
+ * 
+ * TODO: add amount that expenses should be less than
+ * TODO: add tests
```

Comment:
As with earlier comment about TODOs, considering adding an accompanying issue to make sure it happens.

Reply:
yeah, agree. those notes were during a flurry of coding early this week.